### DISCUSSION TITLE
Importation of " make_reduction"

### DISCUSSION THREAD

**Discussion by Yassine97HF** [alan-turing-institute/sktime#3259] [General]
Hello everyone!  i want to import "make_reduction"

`from sktime.forecasting.compose import make_reduction`

i found this problem:
![Screenshot 2022-08-13 174230](https://user-images.githubusercontent.com/102746985/184503175-1f66c84b-46b0-4d97-b332-8db69a66dca6.png)

Please can you help me to fix this error.

PS: i can import any module from sktime library except for those in "sktime.forecasting.compose" with same error.

Thanks!

**Comment by aiwalter** [comment]
Pls provide `sktime` version

**Reply by Yassine97HF** [reply]
Last version  (0.13.1)

**Comment by aiwalter** [comment]
Its hard to make a remote diagnosis without more information. Which OS and Python version it it? Can you try to install it in a new env again?

**Reply by Yassine97HF** [reply]
windows 10, python (3.10) and i try to creat a new env ...  i got  same error .

**Reply by aiwalter** [reply]
I am also on Windows 10 but I cant reproduce your issue.
